client/javascripts/store/configureStore.js

```diff
@@ -12,20 +12,19 @@ import rootReducer from '../reducers';
 const sagaMiddleware = createSagaMiddleware();
 const middlewares = [sagaMiddleware];
 
-let composeEnhancers = compose;
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
```

Comment:
__[eslint]__
* Error - Unexpected dangling '_' in '__REDUX_DEVTOOLS_EXTENSION_COMPOSE__'. (no-underscore-dangle)

<!-- Comment by [SideCI](https://sideci.com) -->